spec/controllers/v0/account_controls_controller_spec.rb

```diff
@@ -0,0 +1,215 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe V0::AccountControlsController, type: :controller do
+  let(:service_account_access_token) { build(:service_account_access_token) }
+  let(:user_account) { create(:user_account) }
+  let!(:logingov_user_verification) { create(:logingov_user_verification, user_account:, locked:) }
+  let(:logingov_uuid) { logingov_user_verification.logingov_uuid }
+  let!(:idme_user_verification) { create(:idme_user_verification, user_account:, locked:) }
+  let(:idme_uuid) { idme_user_verification.idme_uuid }
+  let!(:dslogon_user_verification) { create(:dslogon_user_verification, user_account:, locked:) }
+  let(:dslogon_uuid) { dslogon_user_verification.dslogon_uuid }
+  let!(:mhv_user_verification) { create(:mhv_user_verification, user_account:, locked:) }
```

Comment:
You can probably move all of these `let!` initializations to a `before` block. You are having to initialize them with the bang since you aren't referencing them in your specs.